README.md

```diff
@@ -138,16 +138,16 @@ can also be linked together to form an entire session policy. A linked
 list of `srtp_policy_t` structures is equivalent to a session policy.
 In such a policy, we refer to a single `srtp_policy_t` as an *element*.
 
-An `srtp_policy_t` strucutre contains two `crypto_policy_t` structures
+An `srtp_policy_t` strucutre contains two `srtp_crypto_policy_t` structures
```

Comment:
Could you also change the typo here, while you're at it? `strucutre` -> `structure`
Thanks!

Reply:
Fixed!